docs/release-notes/8.7.asciidoc

```diff
@@ -1,6 +1,53 @@
 [[release-notes-header-8.7.0]]
 == 8.7
 
+[discrete]
+[[release-notes-8.7.1]]
+=== 8.7.1
+
+[discrete]
+[[enhancements-8.7.1]]
+==== Enhancements
+There are no user-facing changes in 8.7.1.
+
+[discrete]
+[[bug-fixes-8.7.1]]
+==== Bug fixes
+* Fixes a bug that caused the {agent} upgrade from 8.6.2 to 8.7.0 to fail for {agents} running the Osquery Manager integration (https://github.com/elastic/elastic-agent/pull/2448[#2448]).
+
++
+
+[NOTE]
+
+=====
+If you are upgrading {agent} to 8.7.1 from 8.6.2 to 8.7.0, the {agent} upgrade might fail for {agents} running the Osquery Manager integration. To prevent this, delete the Osquery Manager integration from your {agents} before upgrading them to 8.7.0, then re-add it after upgrading. 
```

Comment:
This seems to contradict the previous item, which says that we already fixed this bug. So why is the same behavior still happening? Is the idea that we _did_ fix the bug, but users might still have to delete and re-add the integration to get the fix? If that's the case, maybe rephrase this so it doesn't sound identical to the bug fix above. 

Also, should the last sentence be "before upgrading them to 8.7.1" since this is the 8.7.1 section?

Reply:
@joepeeples @kevinlog does the following revision work? It emphasizes that this is only a problem for users on 8.6.2 or 8.7.0, but that it's preventable.

```suggestion
To prevent upgrade issues while upgrading {agent} from 8.6.2 or 8.7.0, delete the Osquery Manager integration from your {agents} before upgrading them, then re-add it after upgrading. 

```

Reply:
LGTM, thanks!

Reply:
@nastasha-solomon this looks good to me, thanks!